src/Analyzers/Core/Analyzers/AbstractCodeQualityDiagnosticAnalyzer.cs

```diff
@@ -48,16 +48,13 @@ public bool OpenFileOnly(SimplifierOptions? options)
             bool isUnnecessary,
             bool isEnabledByDefault = true,
             bool isConfigurable = true,
-            LocalizableString? description = null)
-#pragma warning disable RS0030 // Do not used banned APIs
-            => new(
+            LocalizableString? description = null) => new(
                     id, title, messageFormat,
                     DiagnosticCategory.CodeQuality,
                     DiagnosticSeverity.Info,
                     isEnabledByDefault,
                     description,
                     helpLinkUri: DiagnosticHelper.GetHelpLinkForDiagnosticId(id),
                     customTags: DiagnosticCustomTags.Create(isUnnecessary, isConfigurable, enforceOnBuild));
-#pragma warning disable RS0030 // Do not used banned APIs
```

Comment:
Also restore this.

Reply:
This should be changed to pragma warning ‘enable’

Reply:
@Neme12 Yes, correct, `restrore` is the correct keyword, sorry my bad. `Enable` is the keyword used in VB.